### PR TITLE
stolostron/backlog#27174 add cluster diagram

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

File: clusters/cluster_lifecycle/cluster_lifecycle_arch.adoc [hunk 1]
```diff
@@ -38,14 +40,6 @@ The components of the cluster lifecycle management architecture include the foll
...
-* The _work agent_ applies the manifest work to the managed cluster. The following permission is automatically created within the Clusterrole to allow the managed cluster to access the hub cluster:
-
-** Allows the agent to send events to the hub cluster 
-
-** Allows the agent to `get` or `update` the `manifestworks` resource
-
-** Allows the agent to update the status of `manifestworks` resource
-
-* When a managed cluster add-on is created in a managed cluster namespace on the hub cluster, its hub controller manifests its agent deployment resources with manifest work. The work agent then applies the manifest work to the managed cluster to deploy the add-on agent.
+* The _work agent_ applies the Managed Cluster Import Controller and the Add-on Hub Controller to the managed cluster. The permission to allow the managed cluster to access the hub cluster is automatically created within the Clusterrole and allows the agent to send events to the hub cluster.
```
Comment (qiujian16) [inline clusters/cluster_lifecycle/cluster_lifecycle_arch.adoc]: The _work agent_ applies the addon agent. Managed Cluster Import Controller and the Add-on Hub Controller are running on the hub.

Comment (cadawson) [reply clusters/cluster_lifecycle/cluster_lifecycle_arch.adoc]: Updated the content. Thanks @qiujian16!

**Comment by cadawson** [comment]
@qiujian16: Does this look ok now? Thanks!

File: clusters/cluster_lifecycle/cluster_lifecycle_arch.adoc [hunk 2]
```diff
@@ -5,10 +5,12 @@ The multicluster engine for Kubernetes uses two main types of clusters: _hub clu
...
 
-When you create a managed cluster by using the {mce}, the cluster is created using the {ocp} cluster installer with the Hive resource. You can find more information about the process of installing clusters with the {ocp-short} installer by reading https://access.redhat.com/documentation/en-us/openshift_container_platform/4.11/html/installing/ocp-installation-overview[OpenShift Container Platform installation overview] in the {ocp-short} documentation.  
+When you create a managed cluster by using the {mce}, the cluster is created using the {ocp} cluster installer with the Hive resource. You can find more information about the process of installing clusters with the {ocp-short} installer by reading https://access.redhat.com/documentation/en-us/openshift_container_platform/4.12/html/installing/ocp-installation-overview[OpenShift Container Platform installation overview] in the {ocp-short} documentation.
```
Comment (swopebe) [inline clusters/cluster_lifecycle/cluster_lifecycle_arch.adoc]: I need to change that description a bit, as well

File: clusters/cluster_lifecycle/cluster_lifecycle_arch.adoc [hunk 3]
```diff
@@ -38,14 +40,6 @@ The components of the cluster lifecycle management architecture include the foll
...
-* The _work agent_ applies the manifest work to the managed cluster. The following permission is automatically created within the Clusterrole to allow the managed cluster to access the hub cluster:
-
-** Allows the agent to send events to the hub cluster 
-
-** Allows the agent to `get` or `update` the `manifestworks` resource
-
-** Allows the agent to update the status of `manifestworks` resource
-
-* When a managed cluster add-on is created in a managed cluster namespace on the hub cluster, its hub controller manifests its agent deployment resources with manifest work. The work agent then applies the manifest work to the managed cluster to deploy the add-on agent.
+* The _work agent_ applies the Add-on Agent to the managed cluster. The permission to allow the managed cluster to access the hub cluster is automatically created within the Clusterrole and allows the agent to send events to the hub cluster.
```
Comment (swopebe) [inline clusters/cluster_lifecycle/cluster_lifecycle_arch.adoc:43]: Add-on Agent? Caps?

**Review by oafischer — APPROVED** [review]

**Review by swopebe — APPROVED** [review]

**Pull request merged**